src/Sulu/Bundle/AdminBundle/Resources/js/containers/Navigation/tests/Navigation.test.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
```

Comment:
Sure.. because I've mocked the whole shit...

Reply:
As in other test we disabled flow

Reply:
This is actually only absolutely necessary in tests for views, because of the `withToolbar` function used there. E.g. in lines like this: https://github.com/sulu/sulu/blob/develop/src/Sulu/Bundle/AdminBundle/Resources/js/views/Datagrid/tests/Datagrid.test.js#L82

I've just tested it, and the only thing it complains about in this file is the `Router`, which could be fixed by actually instantiating it and use `jest.mock` to mock the implementation of the Router, as we already did in a lot of other files.

And we used to disable flow in tests because we thought it is a good idea, but we realized that it is not, because this way it is very easy to mock stuff away, and we don't realize that it is not working anymore when the router changes for some reason.